src/klein/_plating.py

```diff
@@ -161,11 +163,45 @@ def _elementify(self, instance, to_fill_with):
                              boundInstance=instance,
                              presentationSlots=self._presentationSlots)
 
+    @attr.s
+    class _Widget(object):
+        """
+        Implementation of L{Plating.widgeted}.  This is a L{callable}
```

Comment:
Callable, not callable, right?

Reply:
Good point - it needs to implement `__get__`.